src/FFmpegReader.cpp

```diff
@@ -671,8 +671,13 @@ bool FFmpegReader::HasAlbumArt() {
 
 void FFmpegReader::UpdateAudioInfo() {
 	// Set default audio channel layout (if needed)
+#if HAVE_CH_LAYOUT
+	if (!AV_GET_CODEC_ATTRIBUTES(aStream, aCodecCtx)->ch_layout.nb_channels)
```

Comment:
I'm confused by this `if (!nb_channels)` logic... hard coding channel_layout to Stereo if 0 channels are listed? The former logic sets the channel layout if it's not already set, based on the # of channels found. Let's make sure this logic really makes sense, and also, let's try and keep both code paths similar, if possible.